session.go

```diff
@@ -390,6 +391,11 @@ func ParseURL(url string) (*DialInfo, error) {
 			if err != nil {
 				return nil, errors.New("bad value for maxPoolSize: " + opt.value)
 			}
+		case "maxStalenessSeconds":
+			maxStalenessSeconds, err = strconv.Atoi(opt.value)
+			if err != nil {
```

Comment:
I think this would need the `maxStalenessSeconds  >= 0` check to compliment the `SetMaxStalenessSeconds()` behaviour?

Reply:
thanks @domodwyer, I've added you recommendation.